packages/react-percy-ci/src/index.js

```diff
@@ -1,6 +1,7 @@
 import ApiClient from '@percy-io/react-percy-api-client';
 import compileAssets from './compileAssets';
 import createDebug from 'debug';
+import { each } from 'bluebird';
```

Comment:
Maybe we could use https://github.com/yoshuawuyts/promise-each instead of depending on all of bluebird for this? (since bluebird is a huge package)

We currently transitively get bluebird in via percy-js because we rely on `request-promise` so this may not make a large dent, but hopefully at some point we can pull that out and use something lighter.